src/EnergyPlus/PluginManager.cc

```diff
@@ -609,7 +609,7 @@ PluginManager::PluginManager(EnergyPlusData &state) : eplusRunningViaPythonAPI(s
             auto const &fields = instance.value();
             std::string const &thisObjectName = instance.key();
             state.dataInputProcessing->inputProcessor->markObjectAsUsed(sGlobals, thisObjectName);
-            auto const vars = fields.at("global_py_vars");
+            auto const &vars = fields.at("global_py_vars");
```

Comment:
Reminder, `x.at()` returns a reference to a persistent object with the field map. Without `&`, you are making a local copy of that object, which is often not what you want.